tools/ci/push_docs_to_repo.py

```diff
@@ -45,7 +45,6 @@ def run(cmd, stdout=True):
 os.chdir(workdir)
 
 run(['git', 'init'])
-run(['git', 'branch', '-m', 'master', 'main'])
```

Comment:
`git checkout -b main` seems to work after the init.

Reply:
Should we try that first? I am surprised that the "initial" branch would not be named `master` though, so I was wondering if the `git` version mismatch means that renaming branches doesn't really work at all at that time (mainly because at that time my git has `git branch` show nothing).  Do you have an older git version to confirm?

Reply:
> Do you have an older git version to confirm?

No, I have git version 2.29.2 and `git init -b main` also worked for me, but without the specification there were no branches at all. Creating the branch by checking out should be pretty definite. Oddly, a simple `git branch main` fails for lack of a master branch.

Reply:
Oh, sorry, I misread. `git checkout -b main` sounds good, let me do that!

Reply:
Lets try that. Changed it (and mofidied the comment a bit).

Reply:
> Since git version 2.28.0 the git init command now takes a --initial-branch (or -b for short) parameter.

So that isn't an option, but the checkout way seems valid, see https://tinyurl.com/y7v7brcl.